ansible/roles/ocp-workload-dm7-qlb-demo/tasks/workload.yml

```diff
@@ -81,13 +109,25 @@
 #  shell: oc rollout latest dc/rhdm7-qlb-rhdmcentr
 
 - name: "Patch the KIE-Server deployment to use CORS"
-  shell: "oc patch dc/rhdm7-qlb-kieserver --type='json' -p=\"[{'op': 'replace', 'path': '/spec/triggers/0/imageChangeParams/from/name', 'value': 'rhdm73-kieserver-cors:latest'}]\""
+  shell: >-
+    oc patch dc/rhdm7-qlb-kieserver
+    --type='json'
+    -p="[{'op': 'replace', 'path': '/spec/triggers/0/imageChangeParams/from/name',
+    'value': 'rhdm73-kieserver-cors:latest'}]"
 
 - name: "Patch the KIE-Server deployment namespace to use CORS"
-  shell: "oc patch dc/rhdm7-qlb-kieserver --type='json' -p '[{\"op\": \"replace\", \"path\": \"/spec/triggers/0/imageChangeParams/from/namespace\", \"value\": \"{{ocp_project}}\"}]'"
+  shell: >-
+    oc patch dc/rhdm7-qlb-kieserver
+    --type='json'
+    -p '[{"op": "replace", "path": "/spec/triggers/0/imageChangeParams/from/namespace", "value": "{{ ocp_project }}"}]'
```

Comment:
character escapes have been removed, must be tested.

Reply:
Verified that these are equivalent with yaml to json conversion.